src/sagemaker/model_monitor/model_monitoring.py

```diff
@@ -227,6 +227,12 @@ def __init__(
             MONITORING_SCHEDULE_INTER_CONTAINER_ENCRYPTION_PATH,
             sagemaker_session=self.sagemaker_session,
         )
+        self.env = resolve_value_from_config(
```

Comment:
Please evaluate adding unit test coverage for this module?

Reply:
This is also covered in unit-test: test_create_monitoring_schedule_with_sagemaker_config_injection